flutter/pubspec.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  sentry: ">=4.0.0-beta.3 <4.1.0"
+  sentry: ">=4.0.0 <4.1.0"
```

Comment:
We really want to limit to 4.1.0?
Should this be open, or up to 5.0?

Reply:
@bruno-garcia I'm afraid of compatibility as we can't limit to a specific version, at least tighten like this, we know that it's not gonna be broken, for example, we could do `minor` semver changes and it's not gonna be broken unless they upgrade the `sentry_flutter` dependency first, wdyt?

Reply:
yeah honestly it won't matter because we version them together so they'll always get the exact version

Reply:
ok gonna do < 5.0.0